app/src/main/java/com/pulkit4tech/privy/MainActivity.java

```diff
@@ -99,6 +103,15 @@ private void setUpInfo() {
         }
     }
 
+    private void setAddmob() {
+
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'bannerAdView' could be replaced by a local variable.](https://www.codacy.com/app/pulkit4tech/Privy/pullRequest?prid=648686)